src/packagedef.jl

```diff
@@ -86,7 +86,7 @@ function startdebug(socket, error_handler = nothing)
 
                 if msg.cmd == :run
                     try
-                        include(msg.program)
+                        Core.eval(Main, :(include($(msg.program))))
```

Comment:
Can we write this as
```
include_string(Main, msg.program)
```
instead? I'm not sure why/how `include` even works here.

Reply:
`program` is a filename.

Reply:
Ah, that'd do it.
`Main.include(msg.program)` then?

Reply:
Ah, didn't know about that, yes that is better :)